app/models/event.rb

```diff
@@ -0,0 +1,60 @@
+class Event
+
+  attr_reader :name, :start, :end, :description, :location, :link, :thumbnail
+
+  def initialize(sum, start_dt, end_dt, loc, desc, link, thumb)
+    @name = sum
+    @start = start_dt
+    @end = end_dt
+    @location = loc
+    @description = desc
+    @link = link
+    @thumbnail = thumb
+  end
+
+  def self.pull_calendar_data
```

Comment:
Let's go ahead and change this function name for `self.all`, since it is returning all Event objects. Calling `Event.all` reads better.